components/scream/tests/scream_cld_fraction/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+INCLUDE (ScreamUtils)
+
+SET (cld_fractionLib cld_fraction)
```

Comment:
I would remove this intermediate variable. I assume you got the idea from Homme dynamics, but there the issue is that we may build several Homme dynamics libs (they have num levs and qsize hard coded in). Here it's just confusing, I think.